neofetch

```diff
@@ -1068,6 +1068,9 @@ get_distro() {
             # by the shell since the values aren't quoted.
             elif [[ -f /etc/lsb-release && $(< /etc/lsb-release) == *CHROMEOS* ]]; then
                 distro='Chrome OS'
+		
+	    elif [[ -f /etc/vzlinux-release ]]; then
+                distro='VzLinux'
```

Comment:
What happens if this code is removed?

Reply:
```
# cat /etc/os-release 
NAME="Virtuozzo Linux"
```
in os-release it's show Virtuozzo Linux instead of VzLinux

Reply:
Why is os-release not updated to reflect the distribution's actual name?